NAMESPACE

```diff
@@ -7,10 +7,15 @@ export(wedderburn.number, layer.density, lake.number, uStar, water.density,
 	   schmidt.plot, center.buoyancy, ts.center.buoyancy, internal.energy, 
 			 ts.internal.energy, epi.temperature, hypo.temperature, whole.lake.temperature, 
 			 approx.bathy)
+export(depth_filter)
+export(wtr_layer)
```

Comment:
Ok, so I know there's a lot of debate/opinions about using "." separators versus "_" in function names. I don't have a strong opinion either way, but could we keep the exposed exported functions in rLA consistent? So `depth.filter` and `wtr.layer`? That way, this *package* is at least consistent.

Reply:
Yes this is very sensible and really I should have thought about it myself. Addressed here: https://github.com/GLEON/rLakeAnalyzer/pull/85/commits/6fe604cedc3b482103a464565a23f9488d58370c